userspace/libsinsp/threadinfo.cpp

```diff
@@ -68,6 +70,8 @@ libsinsp::state::static_struct::field_infos sinsp_threadinfo::static_fields() co
 	define_static_field(ret, this, m_exe_writable, "exe_writable");
 	define_static_field(ret, this, m_exe_upper_layer, "exe_upper_layer");
 	define_static_field(ret, this, m_exe_from_memfd, "exe_from_memfd");
+	define_static_field(ret, this, m_env_table_adapter.table_ptr(), "args", true);
+	define_static_field(ret, this, m_env_table_adapter.table_ptr(), "env_vars", true);
```

Comment:
```suggestion
	define_static_field(ret, this, m_env_table_adapter.table_ptr(), "env", true);
```

Reply:
Ie: i'd use just `env` here, but that is my own preference.

Reply:
+1 keep it as the names we use in our filters, aka `args` and `env`.